kornia/enhance/adjust.py

```diff
@@ -614,19 +618,25 @@ def _solarize(input: Tensor, thresholds: Union[float, Tensor] = 0.5) -> Tensor:
     if not isinstance(thresholds, (float, Tensor)):
         raise TypeError(f"The factor should be either a float or Tensor. Got {type(thresholds)}")
 
+    if relaxation is not None and relaxation not in ['sigmoid']:
```

Comment:
You can also use the KORNIA_CHECK API here https://github.com/kornia/kornia/blob/master/kornia/core/check.py#L85
```suggestion
    if relaxation is not None and relaxation not in {'sigmoid'}:
```